bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/TextLayout.java

```diff
@@ -42,8 +42,8 @@
  */
 public final class TextLayout extends Resource {
 	Font font;
-	String text, segmentsText;
-	int lineSpacingInPoints;
+	String text, segmentsText, originalText;
+	int lineSpacingInPoints, previousCountNextLine = 0, previousEnd = -1;
```

Comment:
Initializing to 0 is pointless and actually generates more byte code.

Reply:
Updated initialization as part of constructor.